synapseclient/client.py

```diff
@@ -296,20 +296,20 @@ def _init_logger(self):
         logging.getLogger("py.warnings").handlers = self.logger.handlers
 
     @property
-    def max_threads(self):
+    def max_threads(self) -> int:
         return self._max_threads
 
     @max_threads.setter
     def max_threads(self, value: int):
```

Comment:
@thomasyu888 Where do we currently stand with type-hinting void functions?  Are we omitting type hints altogether or is there a plan to use `-> None` (alternatively, `-> NoReturn` from [typing](https://docs.python.org/3/library/typing.html#typing.NoReturn))?

Reply:
From what I am reading `-> None` seems like the appropriate approach.

From typing it seems like it is implying that the function never returns anything.

Take this example:
```
def test_call():
    print("test_call()")

result = test_call()
print(result) # This prints `None`
```

Reply:
Yep! I read that it's only for cases like when a function only throws an exception or something similar.

TIL.